specification/communication/data-plane/role/preview/2022-01-01-preview/CommunicationRole.json

```diff
@@ -0,0 +1,439 @@
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "200": {
+            "description": "OK - Returns the communication role.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationRoleResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a Communication Role": {
+            "$ref": "./examples/GetRole.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CommunicationRoleCreatePermissions": {
+      "description": "The object containing the various types of permissions.",
+      "required": [
+        "dataActions"
+      ],
+      "type": "object",
+      "properties": {
+        "dataActions": {
+          "description": "The list of data oriented action permissions.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CommunicationRolePermissionActions"
+          }
+        }
+      }
+    },
+    "CommunicationRoleCreateRequest": {
+      "required": [
+        "displayName",
+        "isEnabled",
+        "permissions"
+      ],
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "description": "The human readable name of the Role.",
+          "type": "string"
+        },
+        "permissions": {
+          "$ref": "#/definitions/CommunicationRoleCreatePermissions"
+        },
+        "isEnabled": {
+          "description": "Is Role enabled for assignments.",
+          "type": "boolean"
+        }
+      }
+    },
+    "CommunicationRoleGetAllResponse": {
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The query response content",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CommunicationRoleResponse"
+          }
+        },
+        "maxItems": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "@nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "CommunicationRolePermissionActions": {
+      "description": "The available action permissions list.",
+      "enum": [
+        "communication.services/calling/audio/receive/action",
+        "communication.services/calling/video/receive/action",
+        "muteThemselves",
+        "communication.services/calling/mute/others/audio/action",
+        "communication.services/calling/mute/others/video/action",
```

Comment:
What does it mean to "mute video"?

Reply:
As per our first meeting on this topic, the spec document [here](https://microsoft.sharepoint-df.com/teams/IC3SDK/Shared%20Documents/Roles/ACS%20-%20Roles%20and%20Permissions%20-%20Architecture%20Specification.docx?web=1) is the right place to discuss the permissions themselves.
The Swagger only reflects the spec.